mods/ra/rules/aircraft.yaml

```diff
@@ -97,9 +97,16 @@ MIG:
 		HP: 70
 	Armor:
 		Type: Light
+	RevealsShroud@GROUNDED:
```

Comment:
Ah right, I see.  You could perhaps split out a separate `^PlaneVisibility` template?